lib/ansible/playbook/collectionsearch.py

```diff
@@ -60,4 +61,12 @@ def _load_collections(self, attr, ds):
                 display.warning('"collections" is not templatable, but we found: %s, '
                                 'it will not be templated and will be used "as is".' % (collection_name))
 
+            if collection_name in ('ansible.builtin', 'ansible.legacy',):
+                continue
+
+            try:
+                import_module('ansible_collections.' + collection_name)
```

Comment:
Should we maybe use `find_spec` instead?  I'm concerned about what could happen if someone dropped an `__init__.py` in the root of their collection that did something we don't want.

https://docs.python.org/3/library/importlib.html#checking-if-a-module-can-be-imported

Reply:
Yes, good catch.

Reply:
agreed, i would use collections/list or the galaxy collection functions as they go by directory and make no judgement on collection content.